File: O-Maior/OMaior.java
import java.util.Scanner;

public class OMaior{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String valores = in.nextLine();
        String[] numeros = valores.split(" ");
        int MaiorAB = (Integer.parseInt(numeros[0]) + Integer.parseInt(numeros[1]) + Math.abs(Integer.parseInt(numeros[0]) - Integer.parseInt(numeros[1]))) / 2;
        System.out.println(((MaiorAB + Integer.parseInt(numeros[2]) + Math.abs(MaiorAB - Integer.parseInt(numeros[2]))) / 2) + " eh o maior");
    }
}

File: O-Maior/OMaior.cs
using System;
using System.Linq;

public class OMaior{
    public static void Main(){
        int[] valores = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
        int MaiorAB = (valores[0] + valores[1] + Math.Abs(valores[0] - valores[1])) / 2;
        Console.WriteLine($"{(MaiorAB + valores[2] + Math.Abs(MaiorAB - valores[2])) / 2} eh o maior");
    }
}